docs/contributing.md

```diff
@@ -59,7 +59,12 @@ The docs are a collection of [Markdown files][markdown] that also utilize
 For more information on how to use GitDocs read the [GitDocs
 guide][gitdocs_guide].
 
-# Regarding style
+# Regarding language & style
+
+We ask that you avoid trivializing terms when contributing to documentation.
+This includes words like "just", "simply", "easy", "obvious", and
+"straightforward". You can learn more about why we want to avoid this kind of
+language in [this blog post](https://jessitron.com/2020/06/26/just-dont).
```

Comment:
There's actually a DEV post from 2017 which got a ton of reads and really helped kick off this conversation in the first place.

Maybe consider maybe using that one instead, or not if you feel like this one does a better job describing the situation? 🙂

Reply:
@benhalpern no reason to not link to it except that i didn't know that it existed :) do you have a link handy? i can add it in here as well.

Reply:
Oh whoops, I thought I linked to it in that comment @vaidehijoshi 

https://dev.to/marcuscreo/the-4-letter-word-word-that-makes-my-blood-boil

Looks like this got merged and I missed my chance. 😄

I didn't feel super strongly so I'll just say that fate chose.